guides/common/modules/proc_installing-advisor-engine-using-import-and-export.adoc

```diff
@@ -0,0 +1,30 @@
+[id="installing-advisor-engine-using-import-and-export"]
+= Installing {advisorengine} using import and export
+
+You can transfer the container image from a connected system to a disconnected {ProjectServer}. 
+
+.Prerequisites
+* Ensure that `skopeo` and {foreman-installer} are installed and available.
```

Comment:
same as above. For satellite, skopeo is part of the container tools as linked above. For upstream, you can write "{project-package-install} skopeo". Refs https://github.com/containers/skopeo/blob/main/install.md#rhel--centos-stream--8